Cargo.toml

```diff
@@ -4,7 +4,7 @@ description = "Stackable Operator Framework"
 edition = "2018"
 license = "Apache-2.0"
 name = "stackable-operator"
-version = "0.4.0-nightly"
+version = "0.5.0-nightly"
```

Comment:
Why did you bump the version here?
If that's required, it should also be in the changelog

Reply:
No, this is the _next development version_. In the CHANGELOG the `[Unreleased]` line will be replaced with 0.5.0 the next time we make a release.

Reply:
Alright :+1: